docker/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM ubuntu:22.04
 
+ARG TARGETPLATFORM
```

Comment:
```suggestion
ARG TARGET_PLATFORM
```

For better reading?

Reply:
`TARGETPLATFORM` is a built-in argument for docker buildx and cannot be changed.
> https://github.com/docker/buildx#building-multi-platform-images

Reply:
Lesson learnt...